core/src/main/scala/com/lightbend/paradox/markdown/Frontin.scala

```diff
@@ -30,20 +30,20 @@ object Frontin {
 
   def apply(file: File): Frontin = {
     var source: BufferedSource = null
-    val lines = try {
-      source = scala.io.Source.fromFile(file)("UTF-8")
-      source.getLines.mkString("\n")
-    } finally {
-      source.close()
-    }
+    val lines =
+      try {
+        source = scala.io.Source.fromFile(file)("UTF-8")
+        source.getLines.mkString("\n")
+      } finally
+        source.close()
     apply(lines)
   }
 
   def apply(str: String): Frontin =
     str.linesWithSeparators.toList match {
       case Nil => Frontin(Map.empty[String, String], "")
       case x :: xs if separates(x) =>
-        xs span { !separates(_) } match {
+        xs span !separates(_) match {
```

Comment:
So I managed to find an instance where scalafmt changes the meaning of the program and actually causes code not to compile. Will create a ticket upstream.

Reply:
Issue created at https://github.com/scalameta/scalafmt/issues/3182

Reply:
Looks like fix is coming already, but spelling this out fully so it's not broken for now would be good.

Reply:
So I would recommend one out of 2 options.

1. Format the entire codebase and then remove, fix the above problem and then temporarily remove `RedundantBraces` until the next scalafmt release.
2. Just wait for the next scalafmt release. In the rare cases where scalafmt breaks code it typically doesn't take long for a new release.

Reply:
I was thinking just write it in a way that's both formatted and not broken. Either of these should be fine:

```scala
xs.span(!separates(_))
```

```scala
xs span { x => !separates(x) }
```

Reply:
So I realized that `xs span separates` also works and is the most concise/aesthetically pleasing. Changing it now.